.gitbook.yaml

```
@@ -0,0 +1,10 @@
+redirects:
+  guidelines/certification: your-journey/certification.md
+  guidelines/communicate: events/communicate.md
+  use-cases/payment-automation/README: use-cases/payment-automation.md
+  use-cases/payment-automation/using-tokenized-credit-cards: use-cases/payment-automation.md
+  webhooks/README: events/README.md
```

issue: The key must correspond to the path, there's no README in URL (`https://mews-systems.gitbook.io/connector-api/webhooks`)

```suggestion
  webhooks: events/README.md
```